crates/cli/src/args/encrypt.rs

```diff
@@ -28,23 +28,23 @@ impl ConfigArg for EncryptArgs {
 
 impl MergeConfig for EncryptArgs {
     fn merge_config(&mut self, config: Config) {
+        let file_path = self.input_args.file.as_deref().unwrap_or(&Path::new(""));
```

Comment:
Starting to look really good!! 

I think I would like to add a comment here as to why we are falling back to an empty path. Might look out of place otherwise.

Something along the lines of  `// Using empty string as pseudo file name to support creation rules for stdin.`

Reply:
I used the same comment as the `refresh` code path for consistency.